example-app/app/Http/Resources/MovieResource.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Http\Resources;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class MovieResource extends JsonResource
+{
+    /**
+     * Transform the resource into an array.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return array|\Illuminate\Contracts\Support\Arrayable|\JsonSerializable
+     */
+    public function toArray($request)
+    {
+        return [
+            'id' => $this->id,
+            'name' => $this->name,
+            'data' => $this->data,
+            'text' => $this->text,
+            'genres' => $this->genres,
+            'actors' => $this->actors,
+            ];
```

Comment:
форматирование
и неправильно отдаешь жанры и актеров
```suggestion
            'genres' => GenreResource::collection($this->genres),
            'actors' => ActorResource::collection($this->actors),
        ];
```